src/test/java/io/cronitor/client/MonitorWithIdentificationTest.java

```diff
@@ -1,86 +1,82 @@
-package ca.sm360.cronitor.client;
+package io.cronitor.client;
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.runners.MockitoJUnitRunner;
+import org.powermock.modules.junit4.PowerMockRunner;
 
-import java.net.URL;
+// import java.net.URL;
 
 import static org.mockito.Mockito.verify;
+import io.cronitor.client.Command;
 
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(PowerMockRunner.class)
 public class MonitorWithIdentificationTest {
 
     @InjectMocks
     private CronitorClient client = new CronitorClient("customAuthKey");
     @Mock
     private CronitorPinger cronitorPinger;
 
-    private String monitorCode = "customMonitorCode";
+    private String monitorCode = "d3x0c1";
```

Comment:
Just make sure that this code is not linked to any of your real customers, even if it's mocked inside this class test you do not want someone to easily find it.

Reply:
This is our universal dummy endpoint reserved for testing purposes!